src/Mod/Arch/ArchMaterial.py

```diff
@@ -652,7 +652,7 @@ def chooseMat(self, card):
         "sets self.material from a card"
         if card in self.cards:
             import importFCMat
-            self.material = importFCMat.read(self.cards[card])
+            self.material = importFCMat.read(self.cards[card][0], self.cards[card][1])
```

Comment:
```suggestion
            self.material = importFCMat.read(self.cards[card])
```
Missed to revert this change on the last commit